src/manjaro.ipxe

```diff
@@ -9,13 +9,12 @@ goto ${menu}
 set os Manjaro Linux
 menu ${os} Installers
 item --gap Official Releases
-item 17.1.8 ${space} ${os} 17.1.8
+item 17.1.10 ${space} ${os} 17.1.10
 choose manjaro_version || goto manjaro_exit
 
 :manjaro_flavor
 menu ${os} ${manjaro_version} Installers
 item --gap Flavors
-item architect ${manjaro_version} Architect
```

Comment:
Any reason for dropping?

Reply:
Only because there aren't no 17.1.10 for that official

Reply:
Yeah, Architect isn't always updated every version, that's why the iseq for arcitect is in place, if you  can change it to:

```
iseq ${flavor} architect && set ${manjaro_version} 17.1.9 ||
```
And add back in Architect we can see about merging it.

Thanks!

Reply:
Sorceforge has switched to osdn, which doesn't fetch the ISO in the script - even directly

Reply:
Yeah, I see the issue, we'll probably need to dig into that as it's currently using Sweden mirrors which aren't ideal.  It's broken currently anyways so we can move it in and we can open a bug for optimizing that mirror.